### PR TITLE
readthedocs needs a setup.py to successfully build the documentation

### PR DESCRIPTION
**Pull request by dudacgf**
Hi,

sorry about this. I created a minimal setup.py so that readthedocs is ok

### CODE REVIEW

**Comment by dudacgf** [comment]
I'm adding a lot of updates and new files to the documentation. You can take a look at the new version at

https://openvasreporting.readthedocs.io/en/latest/

**Comment by dudacgf** [comment]
I think I'm done now

**Comment by dudacgf** [comment]
now I'm done. really

**Comment by dudacgf** [comment]
now i'm really done. i hope

File: openvasreporting/libs/parsed_data.py [hunk 1]
```diff
@@ -13,10 +13,7 @@
 from .config import Config
 import netaddr
 
-try:
-    from xml.etree import cElementTree as Et
-except ImportError:
-    from xml.etree import ElementTree as Et
+from defusedxml import ElementTree as Et
```
Comment (TheGroundZero) [inline openvasreporting/libs/parsed_data.py:16]: Import unused as all type checks are removed from this file?

**Comment by dudacgf** [comment]
That came from using defusedxml. It doesn't expose ElementTree.Element but
I need to use it in the Vulnerability parser. I don"t know how to fix this
alert, because LGTM thinks I don't use it, but I do

Regards

Em qui., 18 de nov. de 2021 04:53, TheGroundZero ***@***.***>
escreveu:

**Pull request merged**